verification/curator-service/src/controllers/sources.ts

```diff
@@ -7,37 +7,71 @@ import { Source } from '../model/source';
  */
 export const list = async (req: Request, res: Response): Promise<void> => {
     const docs = await Source.find({});
-    res.send(docs);
+    res.json(docs);
 };
 
 /**
  * Get a single source.
  */
-export const get = (req: Request, res: Response): void => {
-    res.sendStatus(501);
-    return;
+export const get = async (req: Request, res: Response): Promise<void> => {
+    const doc = await Source.findById(req.params.id);
+    if (!doc) {
+        res.sendStatus(404);
+        return;
+    }
+    res.json(doc);
 };
 
 /**
  * Update a single source.
  */
-export const update = (req: Request, res: Response): void => {
-    res.sendStatus(501);
-    return;
+export const update = async (req: Request, res: Response): Promise<void> => {
+    try {
+        const source = await Source.findByIdAndUpdate(req.params.id, req.body, {
+            // Return the udpated object.
```

Comment:
nit: sp

Reply:
I promise I tried, but I can't figure out where to put a space here ?? Please let me know what you meant :)